adal/src/androidTest/java/com/microsoft/aad/adal/HashMapExtensionTests.java

```diff
@@ -79,10 +79,6 @@ public void testURLFormDecodeNegative() throws IllegalArgumentException,
         assertNotNull(result);
         assertTrue(result.isEmpty());
 
-        result = (HashMap<String, String>) m.invoke(object, "&a=");
```

Comment:
can you add some values here that will test the new path? also if a test case can trigger the exception that would be great.

Reply:
K. I will add more unit tests today.